src/outputs/parthenon_hdf5.cpp

```diff
@@ -660,6 +662,7 @@ void PHDF5Output::WriteLevelsAndLocs_(Mesh *pm, hid_t file, const HDF5::H5P &pl,
 }
 
 void PHDF5Output::WriteSparseInfo_(Mesh *pm, hbool_t *sparse_allocated,
+                                   const std::vector<int> dealloc_count,
```

Comment:
const ref rather than copy? (not that it'd matter performance wise given the use of this function and I can fix that in the OpenPMD PR which will also touch this function)

Reply:
fixed to be const ref.